compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1023,7 +1023,8 @@ static void mrPeepholes(TR::CodeGenerator *cg, TR::Instruction *mrInstruction)
                for (auto stackMapIter = stackMaps.begin(); stackMapIter != stackMaps.end(); ++stackMapIter)
                   {
                   if (cg->getDebug())
-                     traceMsg(comp, "Adjusting register map %p; removing %s, adding %s due to removal of mr %p\n",
+                     if (comp->getOption(TR_TraceCG))
+                        traceMsg(comp, "Adjusting register map %p; removing %s, adding %s due to removal of mr %p\n",
                              *stackMapIter,
```

Comment:
Previously, the arguments to `traceMsg` were aligned to the `(`. It's probably worth preserving this property for consistency.